dev-libs/gobject-introspection/gobject_introspection-1.55.0.recipe

```diff
@@ -0,0 +1,101 @@
+SUMMARY="Introspection system for GObject-based libraries"
+DESCRIPTION="GObject introspection is a middleware layer between C libraries \
+(using GObject) and language bindings. The C library can be scanned at \
+compile time and generate a metadata file, in addition to the actual native \
+C library. Then at runtime, language bindings can read this metadata and \
+automatically provide bindings to call into the C library."
+HOMEPAGE="https://wiki.gnome.org/Projects/GObjectIntrospection"
+COPYRIGHT="Colin Walters
+	Johan Dahlin
+	Matthias Clasen (girepository)
+	Jürg Billeter (scanner)"
+LICENSE="GNU LGPL v2
+	GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://ftp.gnome.org/pub/GNOME/sources/gobject-introspection/1.55/gobject-introspection-$portVersion.tar.xz"
+CHECKSUM_SHA256="888fba5df3ebfa42ea9f24ad98a667fdc151cf37a07349352804028a5ef0948f"
```

Comment:
@Begasus, sorry for the delay to review this PR!

BTW, 1.56.0 has been released recently. Shall we switch to it? (I've already built it on x86_64.)
```diff
-SOURCE_URI="http://ftp.gnome.org/pub/GNOME/sources/gobject-introspection/1.55/gobject-introspection-$portVersion.tar.xz"
-CHECKSUM_SHA256="888fba5df3ebfa42ea9f24ad98a667fdc151cf37a07349352804028a5ef0948f"
+SOURCE_URI="http://ftp.gnome.org/pub/GNOME/sources/gobject-introspection/${portVersion%.*}/gobject-introspection-$portVersion.tar.xz"
+CHECKSUM_SHA256="0d7059fad7aa5ec50d9678aea4ea139acab23737e9cf9ca0d86c615cecbaa0f8"
```
(This is the checksum for 1.56.0, btw.)

Reply:
configure: error: Package requirements (glib-2.0 >= 2.56.0) were not met:

Do you have a more recent version for glib-2.0 there?

Reply:
Oops, you are right. I forgot to mention that I updated the patchset a little bit:
```diff
 diff --git a/configure.ac b/configure.ac
-index 541a102..2210f08 100644
+index 367f2db..32fbf6c 100644
 --- a/configure.ac
 +++ b/configure.ac
 @@ -128,7 +128,7 @@ GIR_DIR="$EXPANDED_DATADIR/$GIR_SUFFIX"
  AC_SUBST(GIR_DIR)
  AC_DEFINE_UNQUOTED(GIR_DIR, "$GIR_DIR", [Director prefix for gir installation])
  
--PKG_CHECK_MODULES(GLIB, [glib-2.0 >= 2.55.0])
+-PKG_CHECK_MODULES(GLIB, [glib-2.0 >= 2.56.0])
 +PKG_CHECK_MODULES(GLIB, [glib-2.0 >= 2.53.0])
  
  PKG_CHECK_MODULES(GOBJECT, [gobject-2.0])
  PKG_CHECK_MODULES(GMODULE, [gmodule-2.0])
 -- 
 2.16.1
```